docs/remove-a-service.md

```diff
@@ -0,0 +1,79 @@
+
+```bash
+bundle exec rake export:links:all
+```
+
+Then, to retrieve the newly created file use `scp-pull`...
+
+```bash
+gds govuk connect scp-pull -e [integration|staging|production] backend source_file destination_file
+```
+
+For example:
+
+```bash
+gds govuk connect scp-pull -e integration backend /var/apps/local-links-manager/public/data/links_to_services_provided_by_local_authorities.csv ~/Desktop
+```
+
+**Note**: Once stored in a secure location, please remember to delete the downloaded file from your local machine.
+
+## 2. Change the format of the service
+
+We need the service to be a standard plain text page in `publisher`, without a start button and local lookup functionality.
+
+To do this, ask a content designer to change the format of the service in `publisher` from `local-transaction` to `answer`.
+
+**Check** [Should not appear in this list](https://publisher.integration.publishing.service.gov.uk/?format_filter=local_transaction&list=published&string_filter=&user_filter=all) | [Should appear in this list](https://publisher.integration.publishing.service.gov.uk/?list=published&string_filter=&user_filter=all&string_filter=&format_filter=answer)
+
+## 3. Remove the LGSL code from `publisher`
+
+`publisher` stores details about the serivce (LGSL code, name and providing tier) in [`data/local_services.csv`](https://github.com/alphagov/publisher/blob/main/data/local_services.csv). Remove the service from this file, create a pull request and get it approved and merged.
+
+## 4. Remove the `Service` from `local-links-manager`
+
+`local-links-manager` has a `Service` model that stores details about the service, together with it's `Interaction` and providing tier(s). To remove the `Service` record, and all dependant records, run the [service:destroy](../lib/tasks/service.rake) rake task.
+
+```bash
+bundle exec rake service:destroy[LGSL_CODE]
+```
+
+**Check** [integration](https://local-links-manager.integration.publishing.service.gov.uk/services)
+
+## 5. Remove the service from `publisher`
+
+We've [previously removed the LGSL code](#3-remove-the-lgsl-code-from-publisher) from `publisher`. Now we need to remove this service - and any other services not in the [`data/local_services.csv`](https://github.com/alphagov/publisher/blob/main/data/local_services.csv) file.
+
+To do this run the [`local_transactions:remove_old_services`](https://github.com/alphagov/publisher/blob/main/lib/tasks/local_transactions.rake) rake task.
+
+```bash
+bundle exec rake local_transactions:remove_old_services
+```
+
+## 6. Unpublish the service in `publisher`
+
+We need to remove and redirect the old `publisher` `local-transaction` service - now an `answer`.
+
+To do this, ask a content designer to `unpublish` giving them the new redirect url in `publisher`.
```

Comment:
Yeah, seems in this instance we want to keep the existing page and URL and update the content. But for other services it might be unpublished and redirected. 

As in both cases this aspect can be done by a content designer - I'll update the wording in the doc.